activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -411,6 +411,8 @@ def column(name, type, index: nil, **options)
           end
         end
 
+        options[:precision] ||= 6 if type == :datetime && !options.key?(:precision) && @conn.supports_datetime_with_precision?
```

Comment:
Could you make this block form? More than 2 conditions and this line gets very long to read

Reply:
done in 73c30a4e2a3a44ab1467ef7599bbabbb089eb5b9